### DISCUSSION TITLE
How to set local custom fonts?

### DISCUSSION THREAD

**Discussion by sJJdGG** [reactjs/reactjs.org-translation#251] [Team Posts]
Hey everyone
We've faced some questions about font.
We're in the discussion of changing the font for our local translations mainly because the current one isn't nice enough in our eyes and there are already other local fonts that are widely used in our local web community.

Questions:
1. Is it allowed?
2. We've already gotten the promise of a free license from the developer of our picked font on the condition that the font file should not be uploaded to the GitHub repo. How do we encounter this requirement? Is there any already chosen CDN we can use? If not can we use a CDN of our own choice?

**Comment by sorousht** [comment]
I got the confirmation of using a custom font before. I do not see any problems with using a custom CDN as long as working for everyone on different countries.

What's the chosen font btw?

**Comment by sJJdGG** [comment]
IranSans

**Comment by gaearon** [comment]
You can add a script to download the font during deployment. The new site does that too for our own font.
